countmasepath.java

```diff
@@ -0,0 +1,43 @@
+import Java.util.*
+
+public class countmasepath {
+
+	public static void main(String[] args) {
+		Scanner s = new Scanner(System.in);
+		int n = s.nextInt();
+
+		int[][] strg = new int[n+1][n+1];
+		System.out.println(countMazepath(n, n, 0, 0, strg));
+		for(int i=0;i<strg.length;i++) {
+			for(int j=0;j<strg.length;j++) {
+				if(strg[i][j]/10==0) {
+				System.out.print(strg[i][j]+"   ");
+				}else {
+					System.out.print(strg[i][j]+"  ");
+					
+				}
+			}
+			System.out.println();
+		}
```

Comment:
Can you specify the need of this nested loop, they seem to be useless.

Reply:
Also, make an entry in Java folder readme